docker/tests/containerTestCommon.psm1

```diff
@@ -110,7 +110,20 @@ function Test-SkipWindows
 
 function Test-SkipLinux
 {
-    return !((Get-DockerEngineOs) -like 'Alpine Linux*')
+    $os = Get-DockerEngineOs
```

Comment:
As we discussed offline, "Get-DockerEngineOs" returns "Docker for Windows" on Windows Host configured in "Linux containers" mode. (Docker version : 17.12.0-ce-win46 (15048)).
This results in "Test-SkipLinux" returning $true and all tests are skipped.

Reply:
Latest changes work fine on Windows Host configured in "Linux containers".